.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+-   repo: https://github.com/awslabs/git-secrets
+    rev: 80230afa8c8bdeac766a0fece36f95ffaa0be778
+    hooks:
+        - id: git-secrets
```

Comment:
hmm, does this work?  doesn't have to run the git-secrets script like the one in https://github.com/awslabs/git-secrets/blob/master/.pre-commit-hooks.yaml#L4 ?

Reply:
just verified myself.  this, as-is, does indeed work.